src/apiGithub.ts

```diff
@@ -34,16 +34,23 @@ export default class ApiGithub {
             headers: { ...headers },
         };
 
+
         if (body)
             options.body = body;
 
 
+        debugApiGithub(`URL: ${this._baseUrl}${relativePath}`);
+        debugApiGithub(`Options before request: `);
+        debugApiGithub(options);
+
         return got(`${this._baseUrl}${relativePath}`, options);
     }
 
     _handleResponse (response: Response): HandleResponseInterface {
 
-        const result = typeof response.body === 'string' ? JSON.parse(response.body) : '';
+        debugApiGithub(`Response: ${response}`);
+
+        const result = typeof response.body === 'string' ? JSON.parse(response.body) : response.body;
```

Comment:
It's good for now, but I suggest you to take a closer at [Type Guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html). You could add more robust type checks with them by using separate interfaces for completed responses and errors.